### DISCUSSION TITLE
Have timer go negative after it hits 0:00

### DISCUSSION THREAD

**Discussion by calhix** [InfiniTimeOrg/InfiniTime#1618] [Ideas]
### Verification

- [X] I searched for similar feature request and found none was relevant.

### Pitch us your idea!

I'd like the timer to go negative so I know how long it has been since it hit 0:00

### Description

It would be nice that after the timer hits 0:00, a separate screen shows up with the timer counting negative to let you know how long it has been since the timer hit 0:00. This could possibly be integrated with the alarm app as well.

**Comment by FintasticMan** [comment]
There is a PR that does that: #557, but that is severely outdated.

**Comment by Riksu9000** [comment]
@calhix @emosenkis Would this still be useful if the timer continued vibrating, because then you probably wouldn't miss it anyway and I suspect it'd be annoying enough that it would be stopped immediately?

**Comment by emosenkis** [comment]
I imagine that it's far from ideal for this use case and I'd be happy to do a follow-up that allows the user to silence the vibrations or just turn them off after 10 seconds but I'd prefer to keep the PR, which I think is strictly an improvement, focused in the hopes of merging it quickly.

**Comment by Riksu9000** [comment]
I like the idea of having the ringing stop after ~10 seconds and keeping the overtime. I'll give the PR a review soon.

**Comment by vkareh** [comment]
This PR implements this feature. The timer will start the ringing pattern and keep it for 10 seconds. The time elapsed counter will continue for up to one minute, after which it will stop and reset.

I think this is a good compromise between "short unoticeable buzz" and "keep bugging me forever".

**Reply by vkareh** [reply]
aaaaand forgot to link the PR :facepalm: 
#1971

**Comment by vladopajic** [comment]
i don't think this is good usecase for standard timer. maybe some other timer app could have this.
